### PR TITLE
chore: Reuse data loader for analytics

### LINKED ISSUE #8840: fix: Use existing dataloader instead of creating a new one for analytics
The new analytics API creates its own dataloader, which caused a memory leak (#8839).

Today, our [hottest query is a query for the User object](https://app.datadoghq.com/apm/resource/gqlexecutor_6-postgres/pg.query/eb5a3717d1fb7bd6?query=%40_top_level%3A1%20env%3Aproduction%20service%3Agqlexecutor_6-postgres%20operation_name%3Apg.query%20resource_name%3A%22SELECT%20%2A%20FROM%20User%20WHERE%20id%20IN%20%28%20%3F%20%29%22&env=production&spanType=service-entry&topGraphs=latency%3Alatency%2Chits%3Acount%2Cerrors%3Acount%2CbreakdownAs%3Apercentage&traces=qson%3A%28data%3A%28%29%2Cversion%3A%210%29&start=1694811136746&end=1694814736746&paused=false)

Instead of creating a new dataloader, which means an extra hit to our DB, we should instead use the existing dataloader, which has already queried the user object.

### PR DESCRIPTION
**Pull request by Dschoordsch**
# Description

Fixes #8840

## Demo

no demo

## Testing scenarios

- smoke test the app

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/server/graphql/mutations/payLater.ts [hunk 1]
```diff
@@ -32,7 +32,10 @@ export default {
 
     // AUTH
     const viewerId = getUserId(authToken)
-    const meeting = (await r.table('NewMeeting').get(meetingId).run()) as Meeting | null
+    const [meeting, viewer] = await Promise.all([
+      r.table('NewMeeting').get(meetingId).run(), //) as Meeting | null
```
Comment (nickoferrall) [inline packages/server/graphql/mutations/payLater.ts:36]: +1 can remove this comment

File: packages/server/graphql/mutations/payLater.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import {GraphQLID, GraphQLNonNull} from 'graphql'
 import {SubscriptionChannel} from 'parabol-client/types/constEnums'
 import getRethink from '../../database/rethinkDriver'
 import {RValue} from '../../database/stricterR'
-import Meeting from '../../database/types/Meeting'
+//import Meeting from '../../database/types/Meeting'
```
Comment (nickoferrall) [inline packages/server/graphql/mutations/payLater.ts:5]: +1 can remove this

**Review by nickoferrall — APPROVED** [review]
Code looks good and I've tested smoke tested a bunch of core functionality

**Review by tghanken — APPROVED** [review]
Smoke tested events showing up in Amplitude

**Pull request merged**